src/mbgl/style/properties.hpp

```diff
@@ -298,6 +339,23 @@ class Properties {
                                other.template get<Ps>().value))...});
             return result;
         }
+
+        Dependency getDependencies() const noexcept {
```

Comment:
`std::make_tuple` is not `noexcept`.

Reply:
This is evaluated at compile time, though I forgot to put `constexpr` on it.  But I can do it more manually to eliminate any ambiguity.

<img width="500" alt="image" src="https://github.com/maplibre/maplibre-native/assets/71895881/2e4a4437-7c7e-40eb-9dfb-3d0434ad4c2b">

Reply:
Ah OK, but this is one compiler.

I would err on the side of safety in general when it comes to `noexcept`.